x-pack/plugins/apm/public/components/shared/Links/MachineLearningLinks/MLJobLink.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('MLJobLink', () => {
     );
 
     expect(href).toEqual(
-      `/basepath/app/ml#/timeseriesexplorer?_g=(ml:(jobIds:!(myservicename-mytransactiontype-high_mean_response_time)),refreshInterval:(pause:true,value:'0'),time:(from:now%2Fw,to:now-4h))`
+      `/basepath/app/ml#/timeseriesexplorer?_g=(ml%3A(jobIds%3A!(myservicename-mytransactiontype-high_mean_response_time))%2CrefreshInterval%3A(pause%3Atrue%2Cvalue%3A'0')%2Ctime%3A(from%3Anow%252Fw%2Cto%3Anow-4h))`
```

Comment:
why did the encoding here change?

Reply:
i saw it would change after running it thru url.format as a query object. So the updated encoding is actually correct (query param varlues being url encoded), even though both will work.

Reply:
I'll go ahead and update it so the format is how it was before, the browser ends up encoding those characters anyways.